### DISCUSSION TITLE
Welcome to mpich Discussions!

### DISCUSSION THREAD

**Discussion by hzhou** [pmodels/mpich#5801] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by bertie635** [comment]
Bonjour M. je suis nouvelle sur ce forum merci. j'ai un problème j'ai tenté d'installer le Modèle WRF sur ubuntu 22.04 et lors de la configuration de mpich un msg d'erreur est apparu. 
error: The Fortran compiler gfortran will not compile files that call
the same routine with arguments of different types.
comment faire svp merci!

**Reply by hzhou** [reply]
Could up upload the corresponding `config.log`?

**Comment by bertie635** [comment]
voici le fichier que j'ai utilisé ./configure --prefix=$DIR/mpich

**Comment by bertie635** [comment]
je vous remercie de me répondre à l'instant

**Comment by bertie635** [comment]
est ce que je dois utiliser un autre config? si c'est le cas j'utilise lequel svp?

**Reply by hzhou** [reply]
Could you upload your console log (your terminal output) please?

**Comment by bertie635** [comment]
Could up upload the corresponding config.log? peut être que je ne comprends pas bien votre réponse que dois je faire s'il vous plaît?

**Comment by bertie635** [comment]
voici la sortie de mon terminal:  root@bertie-HP-Pro-3130-Microtower-PC:/home/bertie/Build_WRF/LIBRARIES/mpich-3.0.4#

voici l' error: The Fortran compiler gfortran will not compile files that call
the same routine with arguments of different types
